keel-sql/src/main/resources/db/changelog/20200923-add-created-at-to-artifact-versions.yml

```diff
@@ -0,0 +1,14 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-created-at-to-artifact-versions
+      author: lpollo
+      changes:
+        - addColumn:
+            tableName: artifact_versions
+            columns:
+              - column:
+                  name: created_at
+                  type: timestamp(6)
+                  constraints:
+                    nullable: true
```

Comment:
This is nullable because we'll start with a bunch of records that don't have the timestamp when we migrate, and also there's no sensible default we can use. We can look into populating this after the fact with a batch job or something.